packages/flutter/lib/src/widgets/image.dart

```diff
@@ -221,6 +221,14 @@ typedef ImageLoadingBuilder = Widget Function(
   ImageChunkEvent loadingProgress,
 );
 
+/// Signature for the method that is called when loading an image results in an
+/// error.
```

Comment:
include a link to [Image.errorBuilder] in this doc comment?

Reply:
Done.  Reworded it a bit as well to indicate what it does.